src/provisioner/config.py

```diff
@@ -69,3 +76,18 @@ def save(self, config_store):
             raise CortxProvisionerError(
                 errno.EINVAL,
                 f'Error occurred while adding CORTX config information into confstore {e}')
+
+    @staticmethod
+    def _verify_release_info(cm_release=''):
```

Comment:
This function should really move into CortxReleaseInfo class. 
CorxRelease.validate(cm_release) # Returns True or False.

Reply:
Done.